source/oauthsso/google.rst

```diff
@@ -64,7 +64,7 @@ ID settings
 .. note::
    Where can I find my callback URL?
 
-   .. figure:: Pics/oauth-Entra-8.gif
+   .. figure:: images/oauth-sso-google-8.gif
```

Comment:
I'm not 100% sure of this fix, please check.

Reply:
Yep thanks, modifications ok